sdk/core/core-http/package.json

```
@@ -82,7 +82,7 @@
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o core-http-lintReport.html || exit 0",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o core-http-lintReport.html",
```

If the report is clean, I would go with this instead:

```suggestion
    "lint": "eslint package.json api-extractor.json src test --ext .ts",
```